sensor/extrinsic_reflector_based_calibrator/src/extrinsic_reflector_based_calibrator.cpp

```diff
@@ -1051,7 +1087,7 @@ ExtrinsicReflectorBasedCalibrator::matchDetections(
 
 void ExtrinsicReflectorBasedCalibrator::trackMatches(
   const std::vector<std::pair<Eigen::Vector3d, Eigen::Vector3d>> & matches,
-  builtin_interfaces::msg::Time & current_time)
```

Comment:
Adding a reference to represent the result of a method that returns void can be considered a coding smell.
There are times where it can not be avoided, but this is not one of those in my opinion.

Reply:
Commit https://github.com/tier4/CalibrationTools/pull/145/commits/82d940da6f5dfdebeea22fd0649d4526e63a8c7f fix this. 
Now the function returns a boolean value. 
`bool is_track_converged = trackMatches(matches, msg->header.stamp);`